tests/Stoa.test.ts

```diff
@@ -324,48 +298,42 @@ describe ('Test of Stoa API Server', () =>
 
     });
 
-    it ('Test of the path /transaction_received', (doneIt: () => void) =>
+    describe ('Test of pending transactions', () =>
     {
-        let uri = URI(host)
-            .port(port)
-            .directory("transaction_received");
-
-        let url = uri.toString();
-        assert.doesNotThrow(async () =>
+        it ('Store one pending transaction', (doneIt: () => void) =>
```

Comment:
This is not asynchronous either ? It looks like a `before` though.

Reply:
And, of course, it runs asynchronously, strictly speaking.
I maintained the existing method for the following reasons.
After this test is complete, the following tests must be carried out: I realized that I had to wait for the server to fully store the data, not completely after sending all the data. So I had no choice but to use `setTimeout` and `doneIt`.